host/libcontainer_backend.go

```diff
@@ -455,21 +401,110 @@ func (l *LibvirtLXCBackend) Run(job *host.Job, runConfig *RunConfig, rateLimitBu
 	}
 	container.RootPath = rootPath
 
-	log.Info("mounting container directories and files")
-	if err := bindMount(l.InitPath, filepath.Join(rootPath, ".containerinit"), false, true); err != nil {
-		log.Error("error bind mounting .containerinit", "err", err)
-		return err
-	}
-	if err := os.MkdirAll(filepath.Join(rootPath, "etc"), 0755); err != nil {
-		log.Error("error creating /etc in container root", "err", err)
-		return err
-	}
-
-	if err := bindMount(l.resolvConf, filepath.Join(rootPath, "etc/resolv.conf"), false, true); err != nil {
-		log.Error("error bind mounting resolv.conf", "err", err)
-		return err
+	config := &configs.Config{
+		Rootfs: rootPath,
+		Capabilities: []string{
+			"CAP_CHOWN",
```

Comment:
This is actually the full list of capabilities from `/usr/include/linux/capability.h` on Ubuntu trusty.

I did intend to strip it down but I forgot. We could stick with the same list we have been using with `libvirt-lxc` (which blacklisted `CAP_SYS_MODULE`, `CAP_SYS_TIME`, `CAP_MKNOD`, `CAP_AUDIT_CONTROL`, and `CAP_MAC_ADMIN`, see [here](https://github.com/libvirt/libvirt/blob/v1.2.2/src/lxc/lxc_container.c#L1741-L1745)), or do a review of each capability to determine if we want to drop it or not (`libcontainer` provides "good defaults" too, see [here](https://github.com/opencontainers/runc/blob/master/libcontainer/SPEC.md#security))